File: src/main/java/com/upokecenter/text/encoders/EncodingUtf16BE.java
package com.upokecenter.text.encoders;

import java.io.*;
import com.upokecenter.util.*;

import com.upokecenter.text.*;

  public class EncodingUtf16BE implements ICharacterEncoding {
   public ICharacterDecoder GetDecoder() {
      return new EncodingUtf16.Decoder(true);
    }

    public ICharacterEncoder GetEncoder() {
      return new EncodingUtf16.Encoder(true);
    }
  }

File: Text/Encoders/EncodingUtf16BE.cs
using System;
using System.IO;
using PeterO;
using PeterO.Mail;
using PeterO.Text;

namespace PeterO.Text.Encoders {
  internal class EncodingUtf16BE : ICharacterEncoding {
   public ICharacterDecoder GetDecoder() {
      return EncodingUtf16.GetDecoder2(true);
    }

    public ICharacterEncoder GetEncoder() {
      return EncodingUtf16.GetEncoder2(true);
    }
  }
}
